cmd/minikube/cmd/start.go

```diff
@@ -1758,12 +1795,21 @@ $ minikube config unset kubernetes-version`)
 		paramVersion = constants.NewestKubernetesVersion
 	}
 
-	nvs, err := semver.Make(strings.TrimPrefix(paramVersion, version.VersionPrefix))
+	kubernetesSemver := strings.TrimPrefix(strings.ToLower(paramVersion), version.VersionPrefix)
+	if isTwoDigitSemver(kubernetesSemver) {
+		potentialPatch := getLatestPatch(kubernetesSemver)
+		if potentialPatch != "" {
+			kubernetesSemver = potentialPatch
+		} else {
+			return "", ErrKubernetesPatchNotFound
+		}
```

Comment:
```suggestion
		if potentialPatch == "" {
			return "", ErrKubernetesPatchNotFound
		}
		kubernetesSemver = potentialPatch
```
nit: Due to the return, flipping the if can reduce nesting